client/README.md

```diff
@@ -62,3 +62,128 @@ To generate a **class** component, run the following
+```
+
+Now that we have the icons in our library we can use it globally throughout the app. Below I  will show you how to use the icons throughout our app in any component/file.
+
+Make sure to have this import syntax in the component you would like to use an icon in:
+```
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+```
+Now that we have the above syntax, lets call an icon!
+
+Suppose we have a beverage component and we want to use the icons that we added to our library in our `app.js` file. It would look something like this:
+```
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+ 
+export const beverage = () => (
+  <div>
+    <FontAwesomeIcon icon="check-square" />
+    Favorite beverage: <FontAwesomeIcon icon="coffee" />
+  </div>
+)
+```
+Simple right? Well what if we wanted to use the brand icons, lets say we had a gadget component, we will do this:
+```
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+ 
+export const Gadget = () => (
+  <div>
+    <FontAwesomeIcon icon="check-square" />
+    Popular gadgets come from vendors like:
+    <FontAwesomeIcon icon={['fab', 'apple']} />
+    <FontAwesomeIcon icon={['fab', 'microsoft']} />
+    <FontAwesomeIcon icon={['fab', 'google']} />
+  </div>
+)
+```
+What if we wanted to use the regular icons with no background, i.e. the `far` package, we would do the same as above but with far:
+```
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+ 
+export const PersonID = () => (
+  <div>
+    Popular Id cards like:
+    <FontAwesomeIcon icon={['far', 'adress-card']} />
+  </div>
+)
+```
+
+Last things to note:
+- When searching an icon with `fa` remember a drop down list will show a list of icons, its helpful because you will be able to see what is what instead of guessing, see example screenshot below:
+
+<img width="1276" alt="Screen Shot 2019-12-10 at 3 55 56 AM" src="https://user-images.githubusercontent.com/54545904/70514174-1cc7e100-1b01-11ea-9879-2f8de7776586.png">
+
+- Also to note that if you find yourself getting confused on what icon you wnat to choose and you dont know the name or you are overwhelmed by number of options, you can head over to [font awesome](https://fontawesome.com/icons?d=gallery) and search the icon you would like and find the specific name to call there, see example below:
+<img width="1408" alt="Screen Shot 2019-12-10 at 4 01 54 AM" src="https://user-images.githubusercontent.com/54545904/70514643-0d956300-1b02-11ea-88f5-4e43e9c0c59d.png">
+<img width="1420" alt="Screen Shot 2019-12-10 at 4 02 49 AM" src="https://user-images.githubusercontent.com/54545904/70514644-0d956300-1b02-11ea-8378-58fefeeed9f0.png">
+
+### More Details:
+- If you would like to learn more about the react font awesome npm package please visit the [package page](https://www.npmjs.com/package/@fortawesome/react-fontawesome)
```

Comment:
This is all really great, but probably a lot more detail than we need. Can you get this down to like ~500 words? It just needs to explain:

* How to use the component
* How to add a component to the library if we need to

Reply:
Check out Tissa's component on the classnames for some guidance on the documentation:

https://github.com/CotripperPlatform/CoTrip/pull/168#pullrequestreview-329907179